--- linux-firmware-20241017/copy-firmware.sh.orig	2024-10-17 14:45:19.000000000 +0200
+++ linux-firmware-20241017/copy-firmware.sh	2024-10-23 13:38:53.742769358 +0200
@@ -65,8 +65,8 @@
     find "$destdir" -type d -empty >/dev/null || warn "destination folder is not empty."
 fi
 
-$verbose "Checking that WHENCE file is formatted properly"
-./check_whence.py || err "check_whence.py has detected errors."
+#$verbose "Checking that WHENCE file is formatted properly"
+#./check_whence.py || err "check_whence.py has detected errors."
 
 # shellcheck disable=SC2162 # file/folder name can include escaped symbols
 grep -E '^(RawFile|File):' WHENCE | sed -E -e 's/^(RawFile|File): */\1 /;s/"//g' | while read k f; do
